tox.ini

```diff
@@ -0,0 +1,13 @@
+[tox]
+envlist = py{27,34,35,36}, flake8
+skip_missing_interpreters = true
+
+[testenv]
+deps = pytest
+       python-dotenv
+commands = py.test {posargs}
+passenv = VCAP_SERVICES
+
+[testenv:flake8]
+deps = flake8
```

Comment:
@costela after some tests I've install the missing libs for `tox`. 

Merging this PR.